Lesson25/Gokhan/filter.js

```diff
@@ -0,0 +1,422 @@
+// Exercise 1:
+// Given an array of numbers, filter out the even numbers.
+const numbersExercise1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+const oddNumbers = numbersExercise1.filter((number) => number%2 !== 0);
+console.log('Exercise 1 - ', 'Odd Numbers: ', oddNumbers);
+
+
+
+// Exercise 2:
+// Given an array of strings, filter out the strings that contain the letter "a".
+const wordsExercise2 = [
+	'apple',
+	'banana',
+	'car',
+	'dog',
+	'elephant',
+	'flower',
+	'guitar',
+	'house',
+	'island',
+	'jungle',
+];
+
+const containLetter = wordsExercise2.filter((word) => word.includes('a'));
+console.log('Exercise 2 - ', containLetter);
+
+
+// Exercise 3:
+// Given an array of objects representing books, filter out the books with a price less than $10.
+const booksExercise9 = [
+	{
+		title: 'To Kill a Mockingbird',
+		author: 'Harper Lee',
+		genre: 'Fiction',
+		price: 12.99,
+	},
+	{
+		title: '1984',
+		author: 'George Orwell',
+		genre: 'Fiction',
+		price: 10.99,
+	},
+	{
+		title: 'The Great Gatsby',
+		author: 'F. Scott Fitzgerald',
+		genre: 'Fiction',
+		price: 9.99,
+	},
+	{
+		title: 'Pride and Prejudice',
+		author: 'Jane Austen',
+		genre: 'Fiction',
+		price: 7.99,
+	},
+	{
+		title: 'To the Lighthouse',
+		author: 'Virginia Woolf',
+		genre: 'Fiction',
+		price: 14.99,
+	},
+	{
+		title: 'Brave New World',
+		author: 'Aldous Huxley',
+		genre: 'Fiction',
+		price: 11.99,
+	},
+	{
+		title: 'The Catcher in the Rye',
+		author: 'J.D. Salinger',
+		genre: 'Fiction',
+		price: 8.99,
+	},
+	{
+		title: 'Moby-Dick',
+		author: 'Herman Melville',
+		genre: 'Fiction',
+		price: 13.99,
+	},
+	{
+		title: 'The Hobbit',
+		author: 'J.R.R. Tolkien',
+		genre: 'Fantasy',
+		price: 10.99,
+	},
+	{
+		title: "Harry Potter and the Sorcerer's Stone",
+		author: 'J.K. Rowling',
+		genre: 'Fantasy',
+		price: 9.99,
+	},
+];
+
+const booksPrice = booksExercise9.filter((book) => book.price < 10);
```

Comment:
We want to keep books with price above 10 in this exercise.
```suggestion
const booksPrice = booksExercise9.filter((book) => book.price > 10);
```